CMake/sitkUseGTest.cmake

```diff
@@ -33,7 +33,9 @@ function(_sitk_gtest_use_gtest_source)
 
   set(BUILD_GTEST                 ON )
   set(BUILD_GMOCK                 OFF)
-
+  if ("${WIN32}")
+      set(gtest_disable_pthreads      ON CACHE INTERNAL "Disable use of pthreads in gtest")
+  endif()
```

Comment:
Something odd is going on. When I test the release branch (which has an if (MINGW) branch rather than a WIN32), I get the compilation error with GTest. However the problem goes away if I use the WIN32 check.

This seems strange as the compiler appears to be detected as MINGW in the simpleitk cmake files, but perhaps the detection isn't done properly before gtest is built?